### ISSUE TITLE
A question about `knowledge distillation` code

### ISSUE THREAD

**Issue by quaeast** [symanto-research/few-shot-learning-label-tuning#2]
Dear Dr. Thomas Müller:

I have read your papper, the moudel you proposed is really concise and efficient, so I want to used it in my own project. When I read the code in this project I didn't find the code about `knowledge distillation` which you mentioned in section 2.2 and table 4(mpnet (LT-DIST)). Do you have any plan to public this part of code?

**Comment by neosyon** [comment]
Dear  quaeast,

Thank you for the interest in our work. As the knowledge distillation is not part of our original contributions, we do not have plans to release the code used in the method described in Section 2.3. However, can be implemented following the details shared in that section. Its main component uses a regular fine-tuning for the model training. If you have specific questions about the implementation, do not hesitate to contact us.

**Comment by 0ddAstronaut** [comment]
Dear Dr. Thomas Müller:
I have a question regarding the knowledge distillation part. As mentioned in the paper, the label distribution in the silver set produced by the fine-tuned model will be used to generate the label embeddings for the unturned model. I wonder which kind of loss function is used for the label embedding generation. My understanding is that label distribution can be regarded as a soft label or a probability distribution over the set of possible labels...
